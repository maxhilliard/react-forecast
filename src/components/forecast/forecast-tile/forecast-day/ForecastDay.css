.forecastDay {
    composes: forecastDay from '../ForecastTile.css';
    font-size: 30px;
    margin: 10px;
}