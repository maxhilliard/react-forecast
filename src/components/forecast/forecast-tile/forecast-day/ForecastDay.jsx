import React from 'react';
import PropTypes from 'prop-types';

import styles from './ForecastDay.css';

const ForecastDay = ({ day }) => (
    <p className={styles.forecastDay}>
        {day}
    </p>
);

ForecastDay.propTypes = {
    day: PropTypes.string.isRequired,
};

export default ForecastDay;
