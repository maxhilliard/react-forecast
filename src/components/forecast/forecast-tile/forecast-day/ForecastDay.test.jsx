import React from 'react';
import { shallow } from 'enzyme';

import ForecastDay from './ForecastDay';

describe('ForecastDay', () => {
    describe('snapshot', () => {
        it('should match snapshot', () => {
            const wrapper = shallow(<ForecastDay day="mock-day" />);

            expect(wrapper).toMatchSnapshot();
        });
    });
});
