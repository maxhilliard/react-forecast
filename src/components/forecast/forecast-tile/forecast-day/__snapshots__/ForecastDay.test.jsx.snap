// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ForecastDay snapshot should match snapshot 1`] = `
<p
  className="forecastDay"
>
  mock-day
</p>
`;
